run.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+#workaround for azure DNS issue
+
+if [ "$EUID" -eq 0 ]
```

Comment:
no we dont but having multiple search domains shouldn't affect "correct" search domains.